tests/snmpsim/cyberpower.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Power Manager
```

Comment:
Yup. That is correct.

[kitsune@vixen42]/home/kitsune% snmpwalk -O n -c public -v 1 pdu1
.1.3.6.1.2.1.1.1.0 = STRING: Power Manager
.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.3808.1.1.3
.1.3.6.1.2.1.1.3.0 = Timeticks: (436921900) 50 days, 13:40:19.00
.1.3.6.1.2.1.1.4.0 = STRING: Zane C. B-H.
.1.3.6.1.2.1.1.5.0 = STRING: pdu1
.1.3.6.1.2.1.1.6.0 = STRING: Zane's office
.1.3.6.1.2.1.1.7.0 = INTEGER: 72
.1.3.6.1.2.1.2.1.0 = INTEGER: 1
.1.3.6.1.2.1.2.2.1.1.1 = INTEGER: 1
.1.3.6.1.2.1.2.2.1.2.1 = STRING: agent
.1.3.6.1.2.1.2.2.1.3.1 = INTEGER: ethernetCsmacd(6)
.1.3.6.1.2.1.2.2.1.4.1 = INTEGER: 1518
.1.3.6.1.2.1.2.2.1.5.1 = Gauge32: 4288190080
.1.3.6.1.2.1.2.2.1.6.1 = STRING: 0:c:15:40:12:75
.1.3.6.1.2.1.2.2.1.7.1 = INTEGER: up(1)
.1.3.6.1.2.1.2.2.1.8.1 = INTEGER: up(1)
.1.3.6.1.2.1.2.2.1.9.1 = Timeticks: (0) 0:00:00.00
.1.3.6.1.2.1.2.2.1.10.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.11.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.12.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.13.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.14.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.15.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.16.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.17.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.18.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.19.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.20.1 = Counter32: 0
.1.3.6.1.2.1.2.2.1.21.1 = Wrong Type (should be Gauge32 or Unsigned32): Counter32: 0
Error in packet.
Reason: (genError) A general failure occured
Exit 2
[kitsune@vixen42]/home/kitsune%